pkg/policy/policy_service.go

```diff
@@ -56,6 +60,18 @@ func NewPolicyService(OCMClient *ocm.Client, AWSClient aws.Client) PolicyService
 // verify role is RedHat managed
 // verify role has quota to attach the policies
 func (p *policyService) ValidateAttachOptions(roleName string, policyArns []string) error {
+	err := p.ValidateDetachOptions(roleName, policyArns)
+	if err != nil {
+		return err
+	}
+	err = validatePolicyQuota(p.AWSClient, roleName, policyArns)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (p *policyService) ValidateDetachOptions(roleName string, policyArns []string) error {
```

Comment:
Should this method be renamed so that it is more generic?
something like ValidateRoleAndPolicies?

Reply:
change to a  private func validateRoleAndPolicies which used by ValidateAttachOptions and ValidateDetachOptions